globalConfig.json

```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:55183/reviewesModule?","mongoDBName":"reviewesModule"}
\ No newline at end of file
+{"mongoUri":"mongodb://127.0.0.1:51279/reviewesModule?","mongoDBName":"reviewesModule"}
```

Comment:
On windows to end a connection to a server the short cut is `ctrl + c` but if I hit `ctrl + z` the server stops running in the terminal, but still runs on the computer memory and will persist until the computer is shut down. Idk if you are running into the error "server in use" but if you are consider how you close your servers

Reply:
that mongo uri is for the database but i am closing those connections after the seed and after the tests so so far no error, but ill keep in mind if do get errors!